<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lUsuario">
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lClave">
     <property name="text">
      <string>Clave</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="leClave">
     <property name="text">
      <string>1234</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="leUsuario">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>fede</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Boton" name="pbLabel">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Validar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Boton</class>
   <extends>QLabel</extends>
   <header>boton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leUsuario</tabstop>
  <tabstop>leClave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
